meta-rdk-mtk-bpir4/recipes-support/parodus/parodus_1.0.bbappend

```diff
@@ -13,6 +13,7 @@ do_install_append () {
     install -d ${D}${base_libdir_native}/rdk
     install -m 0644 ${S}/devices/broadband/parodus/systemd/parodus.service ${D}${systemd_unitdir}/system
     install -m 0755 ${S}/devices/broadband/parodus/scripts/parodus_start.sh ${D}${base_libdir_native}/rdk
+    sed -i "s/eth0/lan0/g" ${D}${base_libdir_native}/rdk/parodus_start.sh
```

Comment:
can you push this change as part of webpa ticket

Reply:
ok chandra